### PR TITLE
Optimize Image Loading with Blurring

### LINKED ISSUE #42: Add lazy loading for images
Some high res images are loading hella slow. We should implement lazy loading for all images to speed things up

### PR DESCRIPTION
**Pull request by tylerlam1**
Resolves #42.

Note that I didn't so this for SVGs because I still can't figure out how to use nextJS images to optimize that.

Check out the images on the Vercel deployments to see the blurred images in action. Use Incognito to prevent image caching.

### CODE REVIEW

File: components/Timeline.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @next/next/no-img-element */
```
Comment (harsohailB) [inline components/Timeline.jsx:1]: Let's remove this from all places now and enforce it so we always do this blur thing?

**Review by harsohailB — APPROVED** [review]
🔥 🔥 🔥 

This works great! Thanks for doing this. I'm not sure if SVG are possible plus they load up fine for me. It was just the images.

File: components/Timeline.jsx (hunk 1 above)
Comment (tylerlam1) [reply components/Timeline.jsx:1]: Will do

Comment (tylerlam1) [reply components/Timeline.jsx:1]: I can't get the Next Image component to work the way I want for a couple SVGs. Keeping those as imgs. Merging anyways.

**Pull request merged**